bom/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ dependencies {
       "text-minimessage",
       "text-serializer-ansi",
       "text-serializer-gson",
-      "text-serializer-json",
+      // "text-serializer-json",
```

Comment:
Dependency constraints are resolving other outgoing configurations -- like test fixtures -- for some reason, which causes deprecation warnings.